static/library/ranking_component.ts

```diff
@@ -89,32 +131,50 @@ export class DatacommonsRankingComponent extends LitElement {
   @property()
   title!: string;
 
+  // DCID of the statistical variable to compare values for
+  @property()
+  variable!: string;
+
+  // List of DCIDs of the statistical variable to compare values for
+  @property({ type: Array<string>, converter: convertArrayAttribute })
+  variables?: string[];
+
   render(): HTMLElement {
+    const variables = this.variables || [this.variable];
+    const statVarSpec = variables.map((statVar) => {
+      return {
+        denom: "",
```

Comment:
hm, one more unifying thing is we should allow per capita for web components? and other relevant stat var spec things

Reply:
Good idea. Added a perCapita field for the web components. I'll add these in as we update each of the tiles.